src/components/custom/Workspace/Workspace.tsx

```diff
@@ -50,59 +51,84 @@ function Workspace({ ids }: { ids: string[] }) {
         }
     };
 
+    const handleCreateCrewClick = () => {
+        const params = new URLSearchParams(searchParams.toString());
+        params.set('create_mode', 'crew');
+        router.push(`/workspace/crew?${params.toString()}`);
+    };
+
+    // Type guard to check if it's a Crew
+    const isCrew = (item: Work | Crew): item is Crew => {
+        return (item as Crew).root_work !== undefined;
+    };
+
     return (
-        <div className="container mx-auto px-4 py-8 max-w-2xl">
-            <h1 className="text-3xl font-bold mb-6">{type} list</h1>
-            <Tabs
-                value={type}
-                onValueChange={value => handleTypeChange(value)}
-                className="w-full"
+        <div className="relative min-h-screen">
```

Comment:
instead of doing relative & absolute, divide the page into left panel, middle panel & right panel thing.

check figma here: https://www.figma.com/design/oe7TaPnT2OPILk9E4muJnq/Melixir?node-id=4025-258&t=TITxKqmDqmZwblSG-0

Reply:
keep right panel empty for now, add button to left panel. middle panel will remain as it is